docs/collections/documents/tutorials/using-spotcollections.md

```diff
@@ -77,71 +77,148 @@ Let's assume your
 <TabItem value="py" label="Python">
 
 ```py
-  from c8 import C8Client
-  import pprint
-
-  if __name__ == '__main__':
-
-    # Variables - URLs
-    global_url = "gdn.paas.macrometa.io"
-    region_urls = [
-        "gdn-sfo2.prod.macrometa.io",
-        "gdn-us-west1.prod.macrometa.io",
-        "gdn-nyc1.prod.macrometa.io"
-    ]
-
-    # Variables - DB
-    email = "nemo@nautilus.com"
-    password = "xxxxxx"
-    geo_fabric = "_system"
-    collection_name = "accounts"
-    read_query = "FOR account IN accounts RETURN account"
-
-    # Variables - Data
-    data = [
-      {"firstname": "Peter", "lastname": "Parker", "City": "NewYork"},
-      {"firstname": "Bruce", "lastname": "Wayne", "City": "Gotham"},
-      {"firstname": "Clark", "lastname": "Kent", "City": "Manhatten"},
-      {"firstname": "Ned", "lastname": "Stark", "City": "Winterfell"},
-      {"firstname": "Tywin", "lastname": "Lannister", "City": "Kings Landing"},
-    ]
-    pp = pprint.PrettyPrinter(indent=4)
-
-    # Step1: Open connection to GDN. You will be routed to closest region.
-    print("1. CONNECT: federation: {},  user: {}".format(global_url, email))
-    client = C8Client(protocol='https', host=global_url, port=443, email=email, password=password, geofabric=geo_fabric)
-
-    # Step2: Create a collection if not exists
-    print("2. CREATE_COLLECTION: region: {},  collection: {}".format(global_url, collection_name))
-    if client.has_collection(collection_name):
+""" This file is a demo for spot collections """
+import pprint
+from c8 import C8Client
+
+# Variables Access
+URL = "api-gdn.paas.macrometa.io"
+API_KEY = "my API key" #Change this to my API key
+API_KEY_ID = "my API key ID" #Change this to my API key ID
+GEO_FABRIC = "_system"
+
+# Variables - DB
+collection_name = "accounts"
+read_query_name = "read_query"
+read_query = {"query": {"name": read_query_name, "value":
+    "FOR account IN accounts RETURN account"}}
+accounts_keys = "FOR k IN accounts RETURN k._key"
+spot_gf_name = "spot"
+region_urls = [
+    "ap-northeast.paas.macrometa.io",
+    "eu-central.paas.macrometa.io",
+    "eu-west.paas.macrometa.io"
+]
+
+# Variables - Data
+data = [
+    {"_key": "1", "firstname": "Peter", "lastname": "Parker", "City": "NewYork"},
+    {"_key": "2", "firstname": "Bruce", "lastname": "Wayne", "City": "Gotham"},
+    {"_key": "3", "firstname": "Clark", "lastname": "Kent", "City": "Manhatten"},
+    {"_key": "4", "firstname": "Ned", "lastname": "Stark", "City": "Winterfell"},
+    {"_key": "5", "firstname": "Tywin", "lastname": "Lannister", "City": "Kings Landing"}
+]
+pp = pprint.PrettyPrinter(indent=4)
+
+
+# Step1: Open connection to GDN. You will be routed to closest region.
```

Comment:
Put a space between the step and the number
Step 1
Step 2
etc.

Reply:
Thanks, Diana. Done :)